src/main/java/net/homework/webapp/controller/DepartmentController.java

```diff
@@ -0,0 +1,63 @@
+package net.homework.webapp.controller;
+
+import net.homework.webapp.dto.DepartmentDto;
+import net.homework.webapp.service.DepartmentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.CollectionUtils;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/departments")
+@EnableAutoConfiguration
```

Comment:
this annotation is contained already in @SpringBootAnnotation

Reply:
removed